### PR TITLE
DISTURBED: add healtcheck endpoint to troubleshoot GHEs webhooks

### PR DESCRIPTION
**Pull request by bgvozdev**
**What's in this PR?**
- Adding a new public endpoint
 
**Why**
Writing a runbook how to troubleshoot connectivity issues between GHEs and the app. This endpoint is to make sure a webhook can be delivered to the app, thus ruling out network issues.

**Added feature flags**
Yeah nah

### CODE REVIEW

File: src/routes/healthcheck/healthcheck-router.ts [hunk 1]
```diff
@@ -14,4 +14,6 @@ HealthcheckRouter.get("/deepcheck", DeepcheckGet);
 /**
  * healthcheck endpoint to check that the app started properly
  */
-HealthcheckRouter.get("/healthcheck", HealthcheckGet);
+HealthcheckRouter.get("/healthcheck", HealthcheckGetPost);
+// To troubleshoot connectivity between GHEs and the app
+HealthcheckRouter.post("/healthcheck/:uuid", HealthcheckGetPost);
```
Comment (gxueatlassian) [inline src/routes/healthcheck/healthcheck-router.ts:19]: Care the reason of POST instead of GET?

Comment (bgvozdev) [reply src/routes/healthcheck/healthcheck-router.ts:19]: because GHEs sending post requests

**Review by joshkay10 — APPROVED** [review]

File: src/routes/healthcheck/healthcheck-router.ts (hunk 1 above)
Comment (gxueatlassian) [reply src/routes/healthcheck/healthcheck-router.ts:19]: Okay, you need POST in poco settings as well

Comment (joshkay10) [reply src/routes/healthcheck/healthcheck-router.ts:19]: oo good catch, and path by the looks

Comment (rachellerathbone) [reply src/routes/healthcheck/healthcheck-router.ts:19]: Can't we just have:
`HealthcheckRouter.get("/healthcheck/:uuid?", HealthcheckGetPost);`

Comment (rachellerathbone) [reply src/routes/healthcheck/healthcheck-router.ts:19]: Can we chain then?

```
HealthcheckRouter.route("/healthcheck/:uuid?")
  .get(HealthcheckGetPost)
  .post(HealthcheckGetPost);
 ```

Comment (gxueatlassian) [reply src/routes/healthcheck/healthcheck-router.ts:19]: so. many responses on the pr, haha, #TeamWork!!

Comment (bgvozdev) [reply src/routes/healthcheck/healthcheck-router.ts:19]: I'd prefer to keep it simple :)

File: etc/poco/bundle/main.json [hunk 2]
```diff
@@ -8,7 +8,7 @@
         "/deepcheck"
       ],
       "methods": [
-        "GET"
+        "GET", "POST"
```
Comment (gxueatlassian) [inline etc/poco/bundle/main.json:11]: /nit
Separate the GET and POST to limit the scope.

Comment (gxueatlassian) [reply etc/poco/bundle/main.json:11]: /nit
Modify the test as well.

**Review by gxueatlassian — DISMISSED** [review]

File: src/routes/healthcheck/healthcheck-router.ts (hunk 1 above)
Comment (rachellerathbone) [reply src/routes/healthcheck/healthcheck-router.ts:19]: Makes the dream work.

**Review by gxueatlassian — APPROVED** [review]

**Pull request merged**